skyrl-train/docs/tutorials/skypilot.rst

```diff
@@ -0,0 +1,167 @@
+Launching with Skypilot
+=====================================
+
+Skypilot
+-------
+
+`SkyPilot <https://docs.skypilot.co/en/latest>`_ is a system to run, manage,
+and scale AI workloads on any infrastructure. It handles all of the logic of
+provisioning, setting up, and tearing down clusters on any cloud.
+
+Setup Skypilot
+--------------
+
+To setup SkyPilot on AWS
```

Comment:
Does this example only work with AWS? we can say skypilot works with all clouds, but here we provide aws as an example

Reply:
When testing I used first `nebius` instead of `aws` since it was cheaper. The only changes needed for switching between clouds would be potentially the GPU names (its been a while since i tested it but iirc `nebius` didn't have L4s but had L40s)

Reply:
Great! I think then we can just say this instruction works for setting up SkyPilot on any clouds, but we just show one example with AWS